examples/use_intelex.py

```diff
@@ -0,0 +1,222 @@
+"""
+Creating models that are accelerated by Intel(R) Extension for scikit-learn
+--------------------------------------------------------------------------------
+
+This guide demonstrates how Intel(R) Extension for scikit-learn (sklearnex) can
+be used to speed up inference of scikit-learn models.
+We train two simple :class:`KNeighborsClassifier` instances, one with and one
```

Comment:
:class:`KNeighborsClassifier`

should probably be

    :class:`sklearn.neighbors.KNeighborsClassifier`

right now sphinx doesn't link it.